packages/xprog-prensa/news/Teaser/PublishedDate/component.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import { Link } from '../../../primitives/Link'
+import * as S from './styles'
+import * as t from './types'
+
+const formatDate = (date) => {
```

Comment:
Será que cabe um try catch aqui para evitar null pointer exception se o date não for um date válido?